pkg/dom0-ztools/rootfs/bin/eve

```diff
@@ -163,6 +182,13 @@ __EOT__
                      ;;
           esac
           ;;
+ config) case "$2" in
+               mount) mount_partlabel "CONFIG" "$3"
+                     ;;
```

Comment:
If we're adding a mount it would make sense to also add an unmount.

Reply:
Good point, added.